index.html

```diff
@@ -1,38 +1,40 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <title>Calculator</title>
-        <link rel="stylesheet" type="text/css" href="src/css/index.css" />
-    </head>
-    <body>
-        <div id="app">
-            <div class="calculator">
-                <h1 id="total">0</h1>
-                <div class="digits flex">
-                    <button class="digit">9</button>
-                    <button class="digit">8</button>
-                    <button class="digit">7</button>
-                    <button class="digit">6</button>
-                    <button class="digit">5</button>
-                    <button class="digit">4</button>
-                    <button class="digit">3</button>
-                    <button class="digit">2</button>
-                    <button class="digit">1</button>
-                    <button class="digit">0</button>
-                </div>
-                <div class="modifiers subgrid">
-                    <button class="modifier">AC</button>
-                </div>
-                <div class="operations subgrid">
-                    <button class="operation">/</button>
-                    <button class="operation">X</button>
-                    <button class="operation">-</button>
-                    <button class="operation">+</button>
-                    <button class="operation">=</button>
-                </div>
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Calculator</title>
+</head>
+
+<body>
+    <div id="app">
+        <div class="calculator">
+            <h1 id="total">0</h1>
+            <div class="digits flex">
+                <button data-button="digit" class="digit">9</button>
+                <button data-button="digit" class="digit">8</button>
+                <button data-button="digit" class="digit">7</button>
+                <button data-button="digit" class="digit">6</button>
+                <button data-button="digit" class="digit">5</button>
+                <button data-button="digit" class="digit">4</button>
+                <button data-button="digit" class="digit">3</button>
+                <button data-button="digit" class="digit">2</button>
+                <button data-button="digit" class="digit">1</button>
+                <button data-button="digit" class="digit">0</button>
+            </div>
```

Comment:
```suggestion
            <div class="digits flex" data-container-type="digit">
                <button class="digit">9</button>
                <button class="digit">8</button>
                <button class="digit">7</button>
                <button class="digit">6</button>
                <button class="digit">5</button>
                <button class="digit">4</button>
                <button class="digit">3</button>
                <button class="digit">2</button>
                <button class="digit">1</button>
                <button class="digit">0</button>
            </div>
```
(마이너) 하위 엘리먼트가 많아서 일일이 달아주기 어렵다면 위처럼 코드를 작성하고 event.target.closest('div')에 있는 dataset을 가져올수도 있어요